cmd/minikube/cmd/kubectl.go

```diff
@@ -96,7 +96,7 @@ host. Please be aware that when using --ssh all paths will apply to the remote m
 			os.Exit(1)
 		}
 
-		if len(args) > 1 && args[0] != "--help" {
+		if len(args) > 1 && args[0] != "--help" && args[0] != "__complete" {
```

Comment:
Thanks for fixing this issue, @ben-krieger 
I am curious, 
can we guarantee the __complete would always be args[0] ? if not maybe we can loop over the args and flag it  as skip dash cluster... ?

Reply:
So put another way, "Are there any valid flags for `kubectl` when using the `__complete` subcommand?" That's a good question. I assumed not, but I'll check the docs/code. Then there's the question of whether we think that any such flags may get added in the future...

Reply:
The global flags are `namespace`, `context`, `cluster`, and `user` (https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/cmd.go#L499-L520). None of those make sense as flags to `__complete`.

But I guess more importantly, `__complete` is a plugin (I guess?) and for the plugin handler code https://github.com/kubernetes/kubectl/blob/31af32762da0400245ada0c342327da4b78eb182/pkg/cmd/cmd.go#L227-L239 it guarantees that there are no flags before the plugin name.

I'm not familiar enough with the plugin architecture, but this seems like a strong guarantee that `__complete` will always be `args[0]`.

Reply:
ok thats fair enough for me ! and thanks for this contribution ! I look forward to see more contributions from you